metadata/lang/es.yml

```diff
@@ -0,0 +1,50 @@
+# metadata box
+overview: "Descripción General"
+questions: "Preguntas"
+objectives: "Objectivos"
+requirements: "TOTRANSLATE"
+authors: "Autores"
```

Comment:
RIGHT? Sadly it defaults to male assumption. But I'll check if there's a cooler version these days than Spanish 101 told us...

Reply:
Usually, the generic plural is the masculine `autores` (neglecting the rest), and a possible solution is to duplicate `autores y autoras`. A generic plural including everyone would be `autoría` (authorship). Another option is to use feminine plural (`autoras`) since this is listing people and "persona" is feminine ;)